docs/en/operation/Metastore.md

```diff
@@ -73,6 +73,35 @@ and `Master.RocksInodeEstimatedMemUsage` estimate the total memory usage for the
 These two metrics are further combined in `Master.RocksTotalEstimatedMemUsage` to estimate the total memory usage of RocksDB across
 all of Alluxio.
 
+### RocksDB configuration
```

Comment:
which configurations do we think will be the most commonly used ones? can we briefly introduce them?